content/projects/data-science-specific/statistical-thinking/_index.md

```diff
@@ -42,38 +42,53 @@ Complete the DataCamp courses [Statistical Thinking in Python Part 1](https://ww
-
-Import the [MDG data](MDG_Export_20191227.csv) to a Jupyter Notebook. You will need the packages `matplotlib`, `numpy`, `seaborn`, `pandas` and `scipy`.
+The Millennium Development Goals were a set of 8 goals for 2015 that were defined by the United Nations to help improve living conditions and the conditions of our planet. Key indicators were defined for each of these goals, to see whether they were being met. We will have a look at some of the key indicators from _Goal 7: Ensure environmental sustainability_, namely carbon dioxide (CO2) emissions, protected land and sea areas, and forests. The full dataset can be found at http://mdgs.un.org/.Your repo should contain everything needed to replicate your work. It's good practice to structure your files well, so we'll expect you to have a separate directory for "data" and "notebook", so that your final file structure looks something like this: 
+
+```
+├──data
+│   └──MDG_Export_20191227.csv
+├──notebook
+│  └──statistical_thinking.ipynb
+├──README.md
+├──requirements.txt
+└──.gitignore 
+```
 
 ### Questions
 
-1. How many different countries are represented? How many missing values are there by country, year and series?
-2. Who are the top and bottom 5 countries in terms of CO2 emissions in 1990 and what are their emissions? How has this changed by 2011?
-
-  - Do you spot any problems with the data? If so, fix it.
-
-3. Calculate the mean and median CO2 emissions for 1990. Why do you think these values differ?
-4. Calculate the minimum, maximum and interquartile range of the CO2 emissions for 1990. Using this information, and the mean and median, what does this tell you about the distribution of CO2 emissions?
-5. Create a histogram of the CO2 emissions for 1990. Is this what you expected from your answers in questions 3 and 4?
-6. Calculate the standard deviation and standard error of the mean for CO2 emissions in 1990. How is the standard error different from the standard deviation?
-7. Create a line graph to show CO2 emissions in Brazil, Russia, China, India, the USA and South Africa over time. What does the graph tell you about the difference and change in C02 emissions in these countries?
-8. What is the mean and standard deviation for land area covered by forest in 1990? Why do you think the standard deviation is so large?
-9. Create histograms for land area covered by forest and percentage of area protected in 1990. Describe the distributions.
-10. Create a scatterplot with a regression line using `seaborn.regplot` to show the relationship between the proportion of land area covered by forest and the percentage of area protected in 2000.
+1. Load the [MDG data](MDG_Export_20191227.csv) into a pandas DataFrame. You will need the packages `matplotlib`, `numpy`, `seaborn`, `pandas` and `scipy`.
```

Comment:
Why do we need to give them all this info now? Did they get it wrong before or is it needed? Smells like spoonfeeding

Reply:
I think at this stage of the curriculum it's fine. We don't do it again after this I don't think and we do specify they work specific things out/plot specific things with those libraries